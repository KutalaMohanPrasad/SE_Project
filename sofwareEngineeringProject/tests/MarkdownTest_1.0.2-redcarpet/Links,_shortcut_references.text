This is the [simple case].

[simple case]: /simple



    This one has a [line
    break].   TODO

    This one has a [line 
    break] with a line-ending space.   TODO

[line break]: /foo


[this] [that] and the [other]

[this]: /this
[that]: /that
[other]: /other
