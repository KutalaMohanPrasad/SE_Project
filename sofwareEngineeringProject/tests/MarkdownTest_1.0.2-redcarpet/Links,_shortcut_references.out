<p>This is the <a href="/simple">simple case</a>.</p>

<pre><code>This one has a [line
break].   TODO

This one has a [line 
break] with a line-ending space.   TODO
</code></pre>

<p><a href="/that">this</a> and the <a href="/other">other</a></p>